OpenOversight/tests/conftest.py

```diff
@@ -131,12 +131,17 @@ def generate_officer():
     )
 
 
-def build_assignment(officer: Officer, units: List[Unit], jobs: Job):
+def build_assignment(officer: Officer, units: List[Optional[Unit]], jobs: Job):
+    unit = random.choice(units)
+    if unit:
+        unit_id = unit.id
+    else:
+        unit_id = None
```

Comment:
Small simplification
```suggestion
    unit_id = unit.id if unit else None
```